tests/et/light_HassTurnOn.yaml

```diff
@@ -0,0 +1,16 @@
+language: et
+tests:
+  - sentences:
+      - "lülita sisse kõik tuled toas elutuba"
+      - "lülita sisse tuli toas elutuba"
+      - "lülita sisse tuled toas elutuba"
+      - "lülita sisse kõik toa elutuba tuled"
+      - "lülita sisse toa elutuba tuli"
+      - "lülita sisse toa elutuba tuled"
+      - "lülita kõik tuled toas elutuba sisse"
+      - "lülita tuled toas elutuba sisse"
+      - "lülita tuli toas elutuba sisse"
+    intent:
+      name: "HassTurnOn"
+      slots:
+        area: elutuba
```

Comment:
These tests are for `homeassistant_HassTurnOn`. Tests in the `light_HassTurnOn`. domain should trigger the sentences in `light_HassTurnOn`, which contain the `light` domain.